runtime/internal/weakref/defaultpool.go

```diff
@@ -0,0 +1,8 @@
+//go:build !safepool
+
+package weakref
```

Comment:
Considering the clone pool isn't actually a weakref pool, we should rename this package.

Reply:
Potentially just calling it pool(s)?

Reply:
Technically it's a weakref pool that won't give you weakrefs :). But yeah, there could be a better name.  I think pool/pools is a bit generic, will try to think about a good one.

Reply:
I've renamed it to `luagc`.